<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>systemUI</class>
 <widget class="QWidget" name="systemUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0,0,0,0);
background-image:url(://Service_Setup/Service_Setup/background.png);
border: 0px;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="serviceSetupDiagnosticLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">FACTORY HARDWARE SYSTEM SETUP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>65</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://transparent.png);
background-color:rgb(0,0,0,60);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="serviceSetupDiagnosticLabel_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>5</y>
       <width>216</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,0);
color:rgb(255,255,255,100);</string>
     </property>
     <property name="text">
      <string notr="true">GANTRY MODEL</string>
     </property>
    </widget>
    <widget class="QLabel" name="gantry">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>5</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,50);
color:rgb(255,255,255,100);</string>
     </property>
     <property name="text">
      <string notr="true">ANALOGIC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRightGantry">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>0</y>
       <width>66</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image: url(:/Service_Setup/Service_Setup/scroll_right.png);
background-color:rgb(0,0,0,0
);</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonLeftGantry">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>0</y>
       <width>66</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image: url(:/Service_Setup/Service_Setup/scroll_left.png);
background-color:rgb(0,0,0,0
);</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>120</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://transparent.png);
background-color:rgb(0,0,0,60);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="serviceSetupDiagnosticLabel_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>5</y>
       <width>236</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,0);
color:rgb(255,255,255,100);</string>
     </property>
     <property name="text">
      <string notr="true">ANODE STARTER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonLeftStarter">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>0</y>
       <width>66</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image: url(:/Service_Setup/Service_Setup/scroll_left.png);
background-color:rgb(0,0,0,0
);</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="starter">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>5</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,50);
color:rgb(255,255,255,100);</string>
     </property>
     <property name="text">
      <string notr="true">HIGH SPEED STARTER</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRightStarter">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>0</y>
       <width>66</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image: url(:/Service_Setup/Service_Setup/scroll_right.png);
background-color:rgb(0,0,0,0
);</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>175</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://transparent.png);
background-color:rgb(0,0,0,60);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="serviceSetupDiagnosticLabel_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>5</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,0);
color:rgb(255,255,255,100);</string>
     </property>
     <property name="text">
      <string notr="true">ROTATION</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRightRotation">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>0</y>
       <width>66</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image: url(:/Service_Setup/Service_Setup/scroll_right.png);
background-color:rgb(0,0,0,0
);</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="rotation">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>5</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,50);
color:rgb(255,255,255,100);</string>
     </property>
     <property name="text">
      <string notr="true">MANUAL ROTATION</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonLeftRotation">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>0</y>
       <width>66</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image: url(:/Service_Setup/Service_Setup/scroll_left.png);
background-color:rgb(0,0,0,0
);</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>285</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://transparent.png);
background-color:rgb(0,0,0,60);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="serviceSetupDiagnosticLabel_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>5</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,0);
color:rgb(255,255,255,100);</string>
     </property>
     <property name="text">
      <string notr="true">DETECTOR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRightDetector">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>0</y>
       <width>66</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image: url(:/Service_Setup/Service_Setup/scroll_right.png);
background-color:rgb(0,0,0,0
);</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="detector">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>5</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,50);
color:rgb(255,255,255,100);</string>
     </property>
     <property name="text">
      <string notr="true">SCREENPLUS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonLeftDetector">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>0</y>
       <width>66</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image: url(:/Service_Setup/Service_Setup/scroll_left.png);
background-color:rgb(0,0,0,0
);</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>390</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://transparent.png);
border-image: url(:/Service_Setup/Service_Setup/yes.png);
background-color:rgb(0,0,0,0
);</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>230</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://transparent.png);
background-color:rgb(0,0,0,60);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="serviceSetupDiagnosticLabel_7">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>5</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,0);
color:rgb(255,255,255,100);</string>
     </property>
     <property name="text">
      <string notr="true">TILT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRightTilt">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>0</y>
       <width>66</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image: url(:/Service_Setup/Service_Setup/scroll_right.png);
background-color:rgb(0,0,0,0
);</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="tilt">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>5</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0,50);
color:rgb(255,255,255,100);</string>
     </property>
     <property name="text">
      <string notr="true">MANUAL ROTATION</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonLeftTilt">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>0</y>
       <width>66</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image: url(:/Service_Setup/Service_Setup/scroll_left.png);
background-color:rgb(0,0,0,0
);</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
